libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationViewModel.java

```diff
@@ -339,4 +354,10 @@ private void startNavigation(DirectionsRoute route) {
       navigation.startNavigation(route);
     }
   }
+
+  public interface Callback {
```

Comment:
I guess this could also be package-private.

Reply:
Don't personally like `Callback` name here. Could be confused with `Callback` from OkHttp. It's also kind of vague IMO because it's not giving any information on the context in which is being used.

Reply:
Why are we declaring `Callback` as an inner `interface`? Any reasons in particular (just curious)?